docs/UserGuide.md

```diff
@@ -35,236 +35,205 @@
 > * Commands are not case-sensitive e.g `list` will match `List`  
 
 #### Viewing help : `help`
-Format: `help`
+Format: `help`  
 
 > Help is also shown if you enter an incorrect command e.g. `run`
 
 #### Adding a task: `add`
-Adds a floating task to the task list  
-Format: `add TASK_NAME`
-This will create a floating task without any start time, end time or deadline.
-Examples:
+Adds a task which can be done at anytime.<br>
```

Comment:
nit: instead of `<br>`, it's mor _right_ to just add a double space at the end of a line.

Same for all uses of `<br>`